uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave.scss

```diff
@@ -2,12 +2,29 @@
   margin-right: 5px;
   position: relative;
   top: 1px;
-  width: calc(100% - 75px);
+  width: calc(100% - 80px);
+
+  select {
+    height: 40px;
+  }
+
+  .crm_custom-select__arrow {
+    line-height: 40px;
+    width: 40px;
+  }
 }
 
 leave-request-record-actions {
   .dropdown-menu {
     left: auto;
     right: 0;
   }
+
+  .dropdown-toggle {
+    height: 40px;
+  }
+
+  i.fa-plus {
```

Comment:
What if we add another i.fa-plus somewhere or change the icon? Please add either unique class and assign a style to it or specify the unique parent container for that `i`. Please do not use use `i.fa-plus`

Reply:
I just checked, that the `margin-right`needs to be applied for not only the `fa-plus`, but also for the `fa-plus-circle`. So instead I am using `i.fa` now.